### PR TITLE
CAF: documentation fixes

### PR DESCRIPTION
**Pull request by wdpypere**
Consists mainly out of code block escaping fixes and correcting 1 typo.

### CODE REVIEW

File: src/main/perl/FileWriter.pm [hunk 1]
```diff
@@ -690,10 +690,4 @@ will be closed automatically when it is destroyed:
 This package inherits from C<IO::String>. Check its man page to
 do powerful things with the already printed contents.
 
-=head1 TODO
```
[the remark this answers is not in the document]
Comment (ned21) [reply src/main/perl/FileWriter.pm]: It's also definitely not covered by a commit message which says "CAF: fix pod code escaping"

File: src/main/perl/Process.pm [hunk 2]
```diff
@@ -723,8 +724,4 @@ it. But please, don't use it without a B<good> reason:
 
 It will only work with the C<execute> method.
 
-=head1 SEE ALSO
```
Comment (ned21) [inline src/main/perl/Process.pm]: Remark still valid if the code still wraps LC::Process?

File: src/main/perl/FileWriter.pm (hunk 1 above)
[the remark this answers is not in the document]
Comment (wdpypere) [reply src/main/perl/FileWriter.pm]: @ned21 I was a bit overzealous in removing that, but I do think TODO should not be in a man page/documentation. Would you agree to removing it if I make issues for it on github?

File: src/main/perl/Process.pm (hunk 2 above)
Comment (stdweird) [reply src/main/perl/Process.pm]: it is already mentioned (even in the description), and all LC::process inertaction is now part of a single method. i would also remove this section

File: src/main/perl/FileWriter.pm (hunk 1 above)
[the remark this answers is not in the document]
Comment (stdweird) [reply src/main/perl/FileWriter.pm]: make an issue for it, a TODO is not part of the pod (but a separate commit is a good idea)

[the remark this answers is not in the document]
Comment (wdpypere) [reply src/main/perl/FileWriter.pm]: a separate issue was made: https://github.com/quattor/CAF/issues/269

**Comment by wdpypere** [comment]
@ned21 the commits were separated with reference to the github issue.

**Pull request merged**